src/seed/index.js

```
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+require('dotenv').config();
+require('module-alias/register');
+
+const checkUserExistence = require('./checkUserExistence');
+const SeedUsers = require('~/seed/seedUsers');
+const SeedCategories = require('~/seed/seedCategories');
+const SeedSubjects = require('~/seed/seedSubjects');
+const logger = require('~/logger/logger'); 
+const shouldDropDatabase = process.argv.includes('--drop');
+const MONGO_URI = process.env.MONGODB_URL;
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)